src/System.Security.Cryptography.Csp/tests/RC2CryptoServiceProviderTests.cs

```diff
@@ -27,5 +30,22 @@ public static void RC2KeySize()
                 Assert.Throws<CryptographicException>(() => rc2.KeySize = 128 + 8);
             }
```

Comment:
Do we really not have tests for the 40 bit salt flag, which need to throw on !Windows?  (Unless we can actually make them work)